pnpm-lock.yaml

```diff
@@ -15,7 +15,7 @@ devDependencies:
     version: 5.48.2(eslint@8.32.0)(typescript@5.0.2)
   '@vitest/coverage-istanbul':
     specifier: ^0.30.0
-    version: 0.30.1(vitest@0.30.0)
+    version: 0.30.0(vitest@0.30.0)
```

Comment:
Interesting how a lot of these versions have _decreased_ in this lockfile. I can take a look.

Reply:
cc @emday4prez - in case you ever see this odd change again (where a branch's lockfile has a bunch of package versions behind the main branch), my steps to resolve it are:

1. Reset the lockfile back to the `main` branch
2. Re-run installation

```shell
git checkout main -- pnpm-lock.yaml
pnpm install
```